### ISSUE TITLE
bailador version - does not work when Bailador is installed

### ISSUE THREAD

**Issue by szabgab** [Bailador/Bailador#164]
```
$ bailador version
Failed to open file /home/ubuntu/rakudo-2017.06/install/share/perl6/site/../Bailador/META6.json: No such file or directory
  in sub MAIN at /home/ubuntu/rakudo-2017.06/install/share/perl6/site/resources/574A5FF295023068BE006B06E14E839176F4C980 line 39
  in block <unit> at /home/ubuntu/rakudo-2017.06/install/share/perl6/site/resources/574A5FF295023068BE006B06E14E839176F4C980 line 48

Actually thrown at:
  in sub from-json at /home/ubuntu/rakudo-2017.06/install/share/perl6/site/sources/87AF732055B0123C44B27EBA91DF9A1FF5060EE1 (JSON::Fast) line 399
  in sub MAIN at /home/ubuntu/rakudo-2017.06/install/share/perl6/site/resources/574A5FF295023068BE006B06E14E839176F4C980 line 40
  in block <unit> at /home/ubuntu/rakudo-2017.06/install/share/perl6/site/resources/574A5FF295023068BE006B06E14E839176F4C980 line 48
```

This is actually version 0.0.5

**Comment by Emeric54** [comment]
I am wondering if trying to fetch the version number from the META6 file is the right way. We could imagine a simple file containing the version number in the `ressources` directory, for example.

**Comment by Emeric54** [comment]
Fixed in Bailador 0.0.7, we can close :+1:
